### DISCUSSION TITLE
How to upgrade from an older version of the EFI

### DISCUSSION THREAD

**Discussion by AlienKevin** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#11] [Q&A] [answered]
I installed macOS Big Sur using an older version of the EFI in this repo. Is there a way for me to upgrade to use the new EFI? Do I have to reinstall macOS from scratch?

**Comment by adityabakare** [comment, marked as answer]
No you don't have to reinstall it. Just mount your EFI partition using [MountEFI](https://github.com/corpnewt/MountEFI), and replace your existing EFI folder with the latest one available over here.
